gradle-jdks-setup/src/integrationTest/java/com/palantir/gradle/jdks/setup/GradleJdkInstallationSetupIntegrationTest.java

```diff
@@ -134,28 +132,14 @@ private Path setupGradleDirectoryStructure(Os os) throws IOException {
         // copy the testing script to the working directory
         Files.copy(Path.of("src/integrationTest/resources/testing-script.sh"), workingDir.resolve("testing-script.sh"));
 
-        // copy the certs to the working directory
-        caResources
-                .readPalantirRootCaFromSystemTruststore()
-                .map(AliasContentCert::getContent)
-                .ifPresent(content -> {
-                    try {
-                        Files.write(workingDir.resolve("palantir.crt"), content.getBytes(StandardCharsets.UTF_8));
-                    } catch (IOException e) {
-                        throw new RuntimeException("Failed to write cert", e);
-                    }
-                });
```

Comment:
No longer need this as we just read certs from the system truststore right?

Reply:
yes